template/postcss.config.js

```diff
@@ -0,0 +1,26 @@
+const postcssConfig = {
+  plugins: [
+    require('postcss-import'),
+    require('tailwindcss'),
+    require('postcss-nested'),
+    require('postcss-custom-properties'),
```

Comment:
We use SCSS files which are compile with the `sass-loader`, so the `postcss-import` and `postcss-nested` PostCSS plugins are not required.